docker/flv2mp4-docker-image/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM centos:7
+
+MAINTAINER Hemani Katyal <hkatyal@redhat.com>
+
+RUN yum install epel-release -y && yum update -y && \ 
+    rpm --import http://li.nux.ro/download/nux/RPM-GPG-KEY-nux.ro && \
+    rpm -Uvh http://li.nux.ro/download/nux/dextop/el7/x86_64/nux-dextop-release-0-5.el7.nux.noarch.rpm && \
+    yum install ffmpeg ffmpeg-devel -y
+
+ADD flv2mp4.sh /usr/local/bin/
```

Comment:
I would leave Dockerfile inside this project, so anyone can see how it is created instead of having to navigate through different projects. 

Also I think that lifecycle of the docker image should be different from project, since we could use in other projects in the future, but not strong opinion.

And yes doing the wget of a concrete file might be better to have a reproducible way. The only thing is that then first we need to tag the version then modify the Dockerfile and create the image.